aldryn_django/__init__.py

```diff
@@ -1 +1 @@
-__version__ = '1.11.9.1'
+__version__ = '1.11.10'
```

Comment:
the format is four numbers.
so 1.11.10.1

Reply:
ok